autrainer/models/crnn.py

```diff
@@ -218,204 +212,31 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         return self.cnn(x)
 
 
-# FDYCRNN model
-
-
-class FDYCRNN(AbstractModel):
+class BiGRU(nn.Module):
```

Comment:
I don't think you really need this class as you simply pass the args to `nn.GRU`